README.md

```diff
@@ -206,17 +206,17 @@ Source code layout
 ---
 
 The Redis root directory just contains this README, the Makefile which
-actually calls the real Makefile inside the `src` directory, an example
-configuration for Redis and Sentinel. Finally you can find a few shell
+calls the real Makefile inside the `src` directory and an example
+configuration for Redis and Sentinel. You can find a few shell
```

Comment:
I the previous text mentions these shell scripts, this may be redundant. perhaps changing it to "The shell scripts are used to ..." to explain what the shell scripts are.